src/masternodes/rpc_proposals.cpp

```diff
@@ -596,7 +596,8 @@ UniValue votegovbatch(const JSONRPCRequest &request) {
                              {"masternodeId", RPCArg::Type::STR_HEX, RPCArg::Optional::OMITTED, "The masternode ID, operator or owner address"},
                              {"decision", RPCArg::Type::STR, RPCArg::Optional::OMITTED, "The vote decision (yes/no/neutral)"},
                      }},
-                 },
+                    {"sleepTime", RPCArg::Type::NUM, RPCArg::Optional::OMITTED, "Sets sleeping time for voteGovBatch, by default the value is set to 500ms"}
```

Comment:
Formatting looks off

Reply:
Thought the same at a glance but look okay in the file imo.

https://github.com/DeFiCh/ain/blob/9fcb98c12c504cd3500ff8265a81ae5a22ebde1c/src/masternodes/rpc_proposals.cpp#L599